.github/workflows/verify.yaml

```
@@ -18,9 +18,23 @@ jobs:
     name: Verify
     runs-on: ubuntu-latest
     steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17.x
     - name: Checkout code
       uses: actions/checkout@v2
+      with:
+        path: go/src/github.com/shipwright-io/build
     - name: Run golangci-lint
       uses: golangci/golangci-lint-action@v2
       with:
+        working-directory: go/src/github.com/shipwright-io/build
         args: --timeout=10m
+    - name: Install Counterfeiter
+      run: |
+        go install github.com/maxbrunsfeld/counterfeiter/v6@latest
+    - name: Run verify-generate
+      env:
+        GOPATH: "/home/runner/work/build/build/go"
```

Is it possible to reference [`GITHUB_WORKSPACE`](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables) here?

```suggestion
        GOPATH: "${GITHUB_WORKSPACE}/go"
```

---

@SaschaSchwarze0 not quite possible, as the env var is only available under the runner, so I assume under the `run` block. I did test this and didnt work. See my latest changes, I do use the variable but into another place.